components/builder-notify/src/error.rs

```diff
@@ -1,11 +1,14 @@
-use std::{error,
+use std::{collections::HashMap,
+          error,
           fmt,
           result};
 
 #[derive(Debug)]
 pub enum Error {
     BuilderCore(builder_core::Error),
     NotificationsError(Box<dyn std::error::Error>),
+    HttpClient(reqwest::Error),
+    ApiError(reqwest::StatusCode, HashMap<String, String>),
```

Comment:
as mentioned above I'd make this `WebhookPushError` and I'd make the 2nd arg a `String`. I don't see it used elsewhere.